test/integrationtest/pipelineRun_test.go

```diff
@@ -1,12 +1,12 @@
 // +build e2e
 
-package test
+package integrationtest
 
 import (
 	"testing"
 
-	f "github.com/SAP/stewardci-core/test/framework"
-	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	test "github.com/SAP/stewardci-core/test"
+	f "github.com/SAP/stewardci-core/test/frameworkth/gcp"
 )
```

Comment:
typo?

Reply:
uhh... sorry. Not sure what happened here.